lti_consumer/static/css/student.css

```diff
@@ -0,0 +1,41 @@
+.xblock-student_view.xblock-student_view-lti_consumer h2.problem-header {
```

Comment:
Ah I was going to make a similar comment, so there's another vote for you, @douglashall,  in that particular improvement.